migrations/20220424193304-create_users_table.sql

```diff
@@ -0,0 +1,16 @@
+
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS users
+(
+    id SERIAL,
+    username TEXT NOT NULL,
+    email TEXT,
+    password TEXT NOT NULL,
+    "dateCreated" DATE NOT NULL,
```

Comment:
not sure is this an example or what but i would setup a
 created_at TIMESTAMPTZ 
 updated_at TIMESTAMPTZ 
that can be reused for all tables (i can do it in a pr later)

Reply:
what is the benefits of that approach rather than this one?
There laravel style vs nodejs :D

Reply:
i just mean putting TIMESTAMPTZ not DATE to get all of the data cos. we will need it for the timelines i guess and making pg take care of it

Reply:
Oh right, didn't notice that part. You're right, will update now, also just noticed that I used camel-case, will change to snake-case

Reply:
in that case do this as well https://x-team.com/blog/automatic-timestamps-with-postgresql/

Reply:
that is out of this PR's scope sorry

Reply:
will make a task for it then

Reply:
awesome, thanks